### ISSUE TITLE
scikit-learn 0.22.1で動かない？

### ISSUE THREAD

**Issue by kujirahand** [kujirahand/book-mlearn-gyomu#8]
https://github.com/kujirahand/book-mlearn-gyomu/blob/master/src/ch2/select/selectAlgorithm.py

が動かないとのこと。
30行目でValueErrorが出るそうです。

**Comment by kujirahand** [comment]
まず、バージョンを0.18.2にしてください。

```
pip uninstall scikit-learn
pip install scikit-learn==0.18.2 --ignore-installed
```

そして、shuffleパラメータを削除します。

```
x_train, x_test, y_train, y_test = train_test_split(x, y, test_size = 0.2, train_size = 0.8, shuffle = True)
```
↓
```
x_train, x_test, y_train, y_test = train_test_split(x, y, test_size = 0.2, train_size = 0.8)
```

すると治ります。

**Comment by kujirahand** [comment]
修正済みです。
